### ISSUE TITLE
pth 转 pt 时会失败 

### ISSUE THREAD

**Issue by wangsale** [yanx27/Pointnet_Pointnet2_pytorch#221]
import torch
from  models import  pointnet2_part_seg_msg

def to_categorical(y, num_classes):
    """ 1-hot encodes a tensor """
    new_y = torch.eye(num_classes)[y.cpu().data.numpy(),]
    if (y.is_cuda):
        return new_y.cuda()
    return new_y

model = pointnet2_part_seg_msg.get_model(4, False)

model.eval()
z=torch.load('b2.pth')
model.load_state_dict(z)

example=torch.rand(1, 3, 2048)

label=torch.rand(1, 1)

traced_script_module = torch.jit.trace(model, (example, to_categorical(label, 1)))
traced_script_module.save("b2.pt")

**Comment by wangsale** [comment]
![image](https://user-images.githubusercontent.com/110961731/223890091-d1925416-00de-488d-814d-d5af82af7531.png)
